android/ijkplayer/ijkplayer-java/src/main/java/tv/danmaku/ijk/media/player/AndroidMediaPlayer.java

```diff
@@ -425,6 +425,10 @@ public void onTimedText(MediaPlayer mp, TimedText text) {
             if (self == null)
                 return;
 
+            //To avoid the Meizu or GIONEE crash when the callback was invoked.
+            if (text == null)
+                return;
```

Comment:
It was invoked by an empty text in subtitle and I disagree that just ignore the message.
Empty messages mostly used for reseting, you should send an empty string to upper layer to tell them reset the text label or something else.